.mci.yml

```diff
@@ -79,7 +79,7 @@ functions:
                 cd mongo-c-driver
                 git checkout 1.3.4
                 rm -rf /data/tmp/c-driver-install
-                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align ${cdriver_configure_flags}
+                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align --enable-debug --enable-optimizations --disable-shm-counters --disable-static --disable-dependency-tracking --with-pic --disable-automatic-init-and-cleanup ${cdriver_configure_flags}
```

Comment:
Mostly, yes. Also to make sure that we control the behavior of cleanup programmatically. And a few flags to make the C driver build smaller and leaner.